app/views/proposals/new.html.haml

```diff
@@ -32,8 +32,17 @@
             .col-md-8
               %p
                 Read the <strong>#{link_to 'guidelines', event_path(event.slug)}</strong> to maximize
-                your chance of approval. Refrain from including any information that
-                would allow a reviewer to identify you.
+                your chance of approval.
+
+              %p
+                Once you submit, you will be able to:
+                %ul
+                  %li
+                    <strong>Communicate with the PC</strong>. Please use that channel
+                    if you have special needs, additional links and so forth.
+                  %li
+                    <strong>Add other speakers</strong> to your proposal.
+
```

Comment:
This is how it renders:

![image](https://user-images.githubusercontent.com/332812/32939492-f406b174-cb7f-11e7-9fa2-edaafa6a3df7.png)